spec/src/main/asciidoc/cdi.asciidoc

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (c) 2017 Contributors to the Eclipse Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[restcdi]]
+== MicroProfile Rest Client CDI Support
+
+Rest Client interfaces may be injected as CDI beans.  It is only required to support injection points that are defined by the application, programmatic lookup is not required to work.
+
+Interfaces are assumed to have a scope of `@Dependent` unless there is another scope defined on the interface.
+
+=== Support for MicroProfile Config
+
+For CDI defined interfaces, it is possible to use MicroProfile Config properties to define additional behaviors of the rest interface.  Assuming this interface:
+
+[source, java]
+----
+package com.mycompany.remoteServices;
+
+public interface MyServiceClient {
+    @GET
+    @Path("/greet")
+    Response greet();
+}
+----
+
+The following properties will then be available:
```

Comment:
Do you mean the MyServiceClient provides the following properties? It will be a config source. If so, what is the ordinal for this config source, as these properties might be overridden by other config sources? The ordinal should be configurable as well.

Reply:
Based on the discussion on the call, will rephrase this to be `The values of the following properties will be provided via MicroProfile Config`: